app/views/headache.nunjucks

```diff
@@ -0,0 +1,70 @@
+{% extends '_layouts/content-sidebar-last.nunjucks' %}
+
+{% block local_header %}
+  <div class="local-header--intro">
+    <p>Most headaches will go away on their own and aren’t a sign of
+      something more serious.</p>
+  </div>
+{% endblock %}
+
+{% block main_content %}
+  <div class="section-unit">
```

Comment:
@Demotive this was to replicate the divider that appears in the design version. Is this still required for this page. 

If so, I've used the existing `.section-unit` class but wrapped it using a `div` as an `article` didn't feel right as it doesn't feel as self-contained as the fungal content blocks.